### ISSUE TITLE
Polygon selector with useblit=True - polygon dissapears

### ISSUE THREAD

**Issue by Stassels** [matplotlib/matplotlib#17487]
When I use this example: https://matplotlib.org/3.1.3/gallery/widgets/polygon_selector_demo.html 
and want to use blitting (see line below), the selection polygon dissapears when polygon is finalized.

```python
self.poly = PolygonSelector(ax, self.onselect, useblit=True)
```

**Matplotlib version: 3.1.2 **

  * Operating system: Ubuntu 16.04
  * Matplotlib version:  3.1.2 (from pipy.org)
  * Matplotlib backend (`Qt5Agg'):
  * Python version: 3.7.6
  * Other libraries: PySide2 - 5.14.2

**Comment by QuLogic** [comment]
Possibly somewhat related to #17476, @anntzer?

**Comment by anntzer** [comment]
Probably the same kind of problem, but `active` is not getting toggled and it's not immediately obvious to me where the background update is missing.

**Comment by QuLogic** [comment]
This is also fixed by #17480.
